sdk/eventhubs/azure-messaging-eventhubs/samples/consume-events/consume_events.cpp

```diff
@@ -57,13 +51,11 @@ int main()
   partitionClientOptions.StartPosition.Earliest = true;
   partitionClientOptions.StartPosition.Inclusive = true;
 
-  Log::Stream(Logger::Level::Verbose)
-      << "Creating partition client. Start position: " << partitionClientOptions.StartPosition;
-  Log::Stream(Logger::Level::Verbose)
-      << "Creating partition client. Start position: " << partitionClientOptions.StartPosition;
+  std::cerr << "Creating partition client. Start position: "
```

Comment:
Why is this being written to cerr rather than cout?

Reply:
I changed it.
But the general purpose of `stderr` is not only errors - it is diagnostic messages, such as what is program currently doing normally as well. `std::cout` is for the "result" or "output". It theoretically can be redirected to stdin of the next cli tool in chain. Stuff like "reading file...", "loading textures..." is supposed to go to `stderr`.

From POSIX.1 standard:
```
At program start-up, three streams shall be predefined and need not be opened explicitly: standard input (for reading conventional input), standard output (for writing conventional output), and standard error (for writing diagnostic output).
```
"for writing diagnostic output", does not say "for writing errors".

https://pubs.opengroup.org/onlinepubs/9699919799/

Nevertheless, I updated this file in the PR to use `std::cout` instead of `std::cerr` in all cases.